frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/mirroring-card/mirroring-card.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+import { K8sResourceKind } from '@console/internal/module/k8s/index';
+import { FirehoseResource } from '@console/internal/components/utils/index';
+import { referenceForModel } from '@console/internal/module/k8s/k8s';
+
+import { OCSServiceModel } from '../../../../models';
+import { CEPH_STORAGE_NAMESPACE } from '../../../../constants';
+import '../../common/storage-efficiency/storage-efficiency-card.scss';
+
+export const MirroringItemBody: React.FC<MirroringItemBodyProps> = React.memo(
+  ({ stats, title, isLoaded, error, getStats }) => {
+    const { t } = useTranslation();
+
+    let status: React.ReactElement;
+
+    if (!isLoaded) {
+      status = <div className="skeleton-text ceph-storage-efficiency-card__item-body--loading" />;
+    } else if (error || !stats) {
+      status = (
+        <span className="co-dashboard-text--small text-muted">
+          {t('ceph-storage-plugin~Disabled')}
+        </span>
+      );
+    } else {
+      status = <span className="ceph-storage-efficiency-card__item-text">{getStats()}</span>;
+    }
+    return (
+      <div className="co-inventory-card__item">
+        <div className="ceph-storage-efficiency-card__item-title">{title}</div>
+        <div
+          className="ceph-storage-efficiency-card__item-status"
+          data-test={`${title}-mirroring-card-status`}
+        >
+          {status}
+        </div>
+      </div>
+    );
+  },
+);
+
+const ocsResource: FirehoseResource = {
```

Comment:
Including the details page, "ocsResource" is used in more than one place, better keep it in a common place.

Reply:
ack...